src/tests/ftest/util/cart_ctl_utils.py

```diff
@@ -82,4 +84,8 @@ def run(self):
         """
         self.log.info('Starting CartCtl')
 
+        drpc_dir = os.environ.get("DAOS_AGENT_DRPC_DIR", None)
+        if 'DAOS_AGENT_DRPC_DIR' not in self.env and drpc_dir is not None:
+            self.env['DAOS_AGENT_DRPC_DIR'] = str(drpc_dir)
```

Comment:
We shouldn't need to do this since it's ultimately ran with `run_command`, which uses `process.run`, which uses `os.environ`

Reply:
Removed